internal/preflight/preflight.go

```diff
@@ -90,21 +90,19 @@ type preflightHelperAPI interface {
 }
 
 type preflightHelper struct {
-	client             client.Client
-	registryAPI        registry.Registry
-	buildAPI           build.Manager
-	authFactory        auth.RegistryAuthGetterFactory
-	kernelOsDtkMapping syncronizedmap.KernelOsDtkMapping
+	client      client.Client
+	registryAPI registry.Registry
+	buildAPI    build.Manager
+	authFactory auth.RegistryAuthGetterFactory
 }
 
 func newPreflightHelper(client client.Client, buildAPI build.Manager, registryAPI registry.Registry,
 	authFactory auth.RegistryAuthGetterFactory, kernelOsDtkMapping syncronizedmap.KernelOsDtkMapping) preflightHelperAPI {
```

Comment:
it needs to be removed from function parameters

Reply:
Done, PTAL.